samples/bot-conversation/python/infra/azure.bicep

```diff
@@ -0,0 +1,44 @@
+@maxLength(20)
+@minLength(4)
+@description('Used to generate names for all resources in this file')
+param resourceBaseName string
+
+@description('Required when create Azure Bot service')
+param botAadAppClientId string
+
+param botAppDomain string
+
+@maxLength(42)
+param botDisplayName string
+
+param botServiceName string = resourceBaseName
+param botServiceSku string = 'F0'
+param microsoftAppType string
+param microsoftAppTenantId string
+
+// Register your web service as a bot with the Bot Framework
+resource botService 'Microsoft.BotService/botServices@2021-03-01' = {
+  kind: 'azurebot'
+  location: 'global'
+  name: botServiceName
+  properties: {
+    displayName: botDisplayName
+    endpoint: 'https://${botAppDomain}/api/messages'
+    msaAppId: botAadAppClientId
+    msaAppType: microsoftAppType
+    msaAppTenantId: microsoftAppType == 'SingleTenant' ? microsoftAppTenantId : ''
```

Comment:
Per https://learn.microsoft.com/en-us/azure/bot-service/bot-service-quickstart-registration?view=azure-bot-service-4.0&tabs=userassigned#bot-identity-information, it seems python only supports multi-tenant bots.
Have you tried SingleTenant?
If only multi-tenant bot work,, them we could simplify the bicep and parameters.json.

Reply:
You are right. Changed~